client/signup/main.jsx

```diff
@@ -251,7 +251,11 @@ class Signup extends Component {
 			this.props.flowName !== prevProps.flowName ||
 			this.props.stepName !== prevProps.stepName
 		) {
-			recordSignupStep( this.props.flowName, this.props.stepName );
+			recordSignupStep( this.props.flowName, this.props.stepName, {
```

Comment:
should we also update the call in `componentDidMount`? https://github.com/Automattic/wp-calypso/blob/trunk/client/signup/main.jsx#L244

Reply:
Yeah, you're right. I've also added to `recordSignupStart` and `recordSignupStep` when component did mount.